zsh/configs/keybindings.zsh

```diff
@@ -10,4 +10,5 @@ bindkey "^R" history-incremental-search-backward
 bindkey "^P" history-search-backward
 bindkey "^Y" accept-and-hold
 bindkey "^N" insert-last-word
+bindkey "^B" push-line-or-edit
```

Comment:
I see that we're using <kbd><kbd>^</kbd><kbd>s</kbd></kbd> for our secondary tmux prefix. Do we disable XON/XOFF flow control somewhere in dotfiles that I'm not seeing? If not, how is that prefix working?

<kbd><kbd>^</kbd><kbd>q</kbd></kbd> would also require XON/XOFF.

The default zle bindings have mapped this key to <kbd><kbd>ESC</kbd><kbd>q</kbd></kbd> for emacs mode and nothing for vi mode. We use vi mode. The emacs mode mapping, due to its use of <kbd>ESC</kbd>, would not work in vi mode.